lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/summary/apriori/APrioriSummarizer.java

```diff
@@ -313,8 +311,8 @@ private void countSingles(List<int[]> encoded, double[] countCol, double[] outli
     }
 
     @Override
-    public Explanation getResults() {
-        List<AttributeSet> results = new ArrayList<>();
+    public APExplanation getResults() {
+        List<ExplanationResult> results = new ArrayList<>();
         for (int o = 1; o <= 3; o++) {
```

Comment:
You can choose which ratio you want now, it's defined when you create a summarizer object